src/server/pps/cmds/cmds.go

```diff
@@ -659,6 +682,9 @@ func newPipelineManifestReader(path string) (result *pipelineManifestReader, ret
 func (r *pipelineManifestReader) nextCreatePipelineRequest() (*ppsclient.CreatePipelineRequest, error) {
 	var result ppsclient.CreatePipelineRequest
 	if err := jsonpb.UnmarshalNext(r.decoder, &result); err != nil {
+		if err == io.EOF {
+			return nil, err
+		}
```

Comment:
It's a subtle bug introduced by another PR yesterday.  There is an assumption in our code that `io.EOF` should be returned from this function, but if you look at the code below it wraps the error.